CHANGELOG.unreleased.md

```diff
@@ -35,5 +35,8 @@
 - @xen-orchestra/xapi patch
 - vhd-lib patch
 - xo-server minor
+- xo-server-perf-alert patch
+- xo-vmdk-to-vhd minor
+- xo-web minor
```

Comment:
Why?

Reply:
Well I don’t know, it’s what left from the rebase.

Reply:
After a rebase, a merge, or anything that touches your code, your need to review your PR to be sure it reflect what you want to change on the main code, and if necessary fix it.

Reply:
Understood!